CMakeLists.txt

```diff
@@ -655,15 +659,23 @@ set(ZLIB_PUBLIC_HDRS
     zlib${SUFFIX}.h
 )
 set(ZLIB_PRIVATE_HDRS
+    adler32_p.h
     crc32.h
+    crc32_p.h
     deflate.h
+    deflate_p.h
     functable.h
     gzguts.h
     inffast.h
     inffixed.h
     inflate.h
+    inflate_p.h
     inftrees.h
+    match_p.h
+    memcopy.h
     trees.h
+    zbuild.h
+    zendian.h
     zutil.h
 )
```

Comment:
Missing fallback_builtins.h and trees_p.h, possibly others, not sure.

Reply:
fallback_builtin.h is added via ZLIB_ARCH_HDRS.

Reply:
Added missing trees_p.h